src/proof/proof_manager.h

```diff
@@ -239,6 +239,12 @@ class ProofManager {
   // for SMT variable names that have spaces and other things
   static std::string sanitize(TNode var);
 
+  // wrap term with (p_app ... ) if needed and print
```

Comment:
Maybe instead of "if needed" just say that "if the term is printed as a boolean".